charts/jaeger/templates/es-configmap.yaml

```diff
@@ -0,0 +1,62 @@
+{{ if (and (not .Values.userconfig) .Values.provisionDataStore.elasticsearch) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: user-config
+  namespace: {{ include "jaeger.namespace" . }}
+  labels:
+    {{- include "jaeger.labels" . | nindent 4 }}
+data:
+  user-config.yaml: |
+    service:
+      extensions: [jaeger_storage, jaeger_query, healthcheckv2]
+      pipelines:
+        traces:
+          receivers: [otlp]
+          processors: [batch]
+          exporters: [jaeger_storage_exporter]
+
+    extensions:
+      healthcheckv2:
+        use_v2: true
+        http:
+            endpoint: 0.0.0.0:13133
+
+      jaeger_query:
+        storage:
+          traces: some_storage
+          traces_archive: another_storage
+
+
+      jaeger_storage:
+        backends:
+          some_storage:
+            elasticsearch:
+              index_prefix: {{- .Values.config.extensions.jaeger_storage.backends.elasticsearch.index_prefix | quote | indent 2 }}
```

Comment:
why do you need to use `.Values.config.extensions.jaeger_storage.backends.elasticsearch. index_prefix ` reference here?